txtparser/ini.go

```diff
@@ -20,7 +20,7 @@ const (
 
 type Operator string // The comparison or assignment operator used in an INI file entry
 
-var RegexKeyOperatorValue = regexp.MustCompile(`([\w._-]+)\s*([<=>]+)\s*["']*(.*?)["']*$`) // Break up a line into key, operator, value.
+var RegexKeyOperatorValue = regexp.MustCompile(`([\w.+_-]+)\s*([<=>]+)\s*["']*(.*?)["']*$`) // Break up a line into key, operator, value.
```

Comment:
It looks like there is no test case to verify that items with `+` really are parsed correctly, but maybe I missed this?

Reply:
Oh, you are right. I totally missed to add this test case. Will fix it soon.

Reply:
new commit includes this test case